packages/plugin-compiler/src/test/resources/sample/input/Sample.kt

```diff
@@ -50,17 +50,16 @@ class Sample : RealmObject {
     var doubleListField: RealmList<Double> = RealmList()
     var objectListField: RealmList<Sample> = RealmList()
 
-    // TODO List types - nullable
-//    var nullableStringListField: RealmList<String?> = RealmList()
-//    var nullableByteListField: RealmList<Byte?> = RealmList()
-//    var nullableCharListField: RealmList<Char?> = RealmList()
-//    var nullableShortListField: RealmList<Short?> = RealmList()
-//    var nullableIntListField: RealmList<Int?> = RealmList()
-//    var nullableLongListField: RealmList<Long?> = RealmList()
-//    var nullableBooleanListField: RealmList<Boolean?> = RealmList()
-//    var nullableFloatListField: RealmList<Float?> = RealmList()
-//    var nullableDoubleListField: RealmList<Double?> = RealmList()
-//    var nullableObjectListField: RealmList<Sample?> = RealmList()
+    // Nullable list types - RealmList<RealmObject?> is not supported
```

Comment:
We could have a separate model (generated a-la `unsupported schema argument` test) with this defined to assert the compilation will fail

Reply:
I will address this in https://github.com/realm/realm-kotlin/issues/290